pkg/dao/repository_config_test.go

```diff
@@ -754,7 +754,7 @@ func (suite *RepositoryConfigSuite) TestListFilterMultipleVersions() {
 	assert.Equal(t, quantity, len(response.Data))
 	assert.Equal(t, int64(quantity), count)
 
-	// By setting SortBy to "version asc" we expect the first page versions lengths to be half and half or 1 and 3
+	// By setting the above seed values and SortBy to "version asc", we expect the first page to contain 10 versions of length 1 and 10 versions of length 3
```

Comment:
Fixed this as per @swadely's recommendations from a previous PR. (Had github automerge issues)